e2e-tests/path-prefix/cypress/integration/asset-prefix.js

```diff
@@ -11,10 +11,6 @@ describe(`assetPrefix`, () => {
   })
 
   describe(`runtime`, () => {
-    it(`prefixes preloads`, () => {
```

Comment:
should we check if no preloads exist?

Reply:
Sure! That'd be good to have

Reply:
@wardpeet We are only removing non "app" bundle JavaScript. Do you mean to test if we only have webpack-runtime app etc?

Reply:
we should still remove preloads from app.js

Reply:
👍